packages/callbacks/tizen/inc/tizen_interop_callbacks_plugin.h

```diff
@@ -24,16 +24,19 @@ FLUTTER_PLUGIN_EXPORT void TizenInteropCallbacksPluginRegisterWithRegistrar(
 }  // extern "C"
 #endif
 
+// Information needed to call the callback registered by the user.
 struct CallbackInfo {
-  void *callbackPtr;
-  void *actualUserData;
+  // The function pointer given to TizenInteropCallbacksRegisterWrappedCallback, should be a pointer created in Dart with ffi.Pointer.fromFunction()
+  void *callback_ptr;
```

Comment:
This struct has only a single member. Should the pointer be wrapped with a struct?

Reply:
no practical reason for it, it has additional user_data field before, but now there is no reason for it, I've left it as struct thinking "if it's already a struct I can as well stay that way, wrapping should not introduce any additional cost", but I can remove the struct if you prefer, because it's kind of a legacy from earlier shape of the implementation

Reply:
I just thought it would be cleaner without the wrapping (a callback ID directly maps into a callback pointer), but it's up to you.